src/app/modules/startup/module.nim

```diff
@@ -320,8 +320,17 @@ method emitObtainingPasswordSuccess*[T](self: Module[T], password: string) =
   self.view.emitObtainingPasswordSuccess(password)
 
 method finishAppLoading*[T](self: Module[T]) =
-  self.delegate.finishAppLoading()
   self.delegate.appReady()
+  let currStateObj = self.view.currentStartupStateObj()
+  if not currStateObj.isNil:
+    var eventType = "user-logged in"
```

Comment:
Is this the correct name of the metric? I felt like there's a typo and shouldn't contain the space...

Reply:
huh, you're right, this doesn't feel right, though I copy pasted it from the issue.

yeah, it's probably a typo, the mobile issue does have a `-` https://github.com/status-im/status-mobile/issues/21679

Nice find